po/de.po

```diff
@@ -1098,7 +1098,6 @@ msgstr ""
 "Auch deine Teilnehmer sind herzlich eingeladen, [uns Rückmeldung zu senden][4]!"
 
 #: src/running-the-course.md:55
-#, fuzzy
```

Comment:
What is this `fuzzy` line that you removed in multiple places?

Reply:
The translations that are marked "fuzzy" are not yet used when building the HTML output. They are the either the result of a script like [cloud-translate](https://github.com/google/comprehensive-rust/pull/291) or they come from `msgmerge`, which is used to import new English strings into a translation.

By removing the fuzzy marker, the translation becomes active and will show up in the output.